controller/plugin-templates.php

```diff
@@ -1,24 +1,5 @@
 <?php
```

Comment:
did you check that no single plugin is using these?

Reply:
yes